cmake/cpack_settings.cmake

```diff
@@ -74,4 +74,12 @@ message(STATUS "Final RPM package name: ${FINAL_PACKAGE_NAME}.rpm")
 
 set(CPACK_RPM_FILE_NAME "${FINAL_PACKAGE_NAME}")
 
+# Reproducible builds Macros
```

Comment:
Adding some comments to describe what each of those does would be really helpful